service/certconfig/v2/resources/vaultrole/create.go

```diff
@@ -1,4 +1,4 @@
-package vaultrolev1
+package vaultrole
```

Comment:
Again, note the v1 vault role resource is now part of the v2 resources to have that aligned.